JS/Lesson32/saffet/my-app/src/components/RecipeDetail/index.jsx

```diff
@@ -0,0 +1,15 @@
+export const RecipeDetail = ({recipe}) => {
+
+    console.log(recipe);
```

Comment:
You don't need to render component at all if no recipe data is passed to it. Instead, add a check in the RecipeList (see my comment there).